pkg/oci/ccm.go

```diff
@@ -235,9 +235,9 @@ func buildConfigurationProvider(logger *zap.Logger, config *Config) (common.Conf
 	return cp, nil
 }
 
-// newRateLimiter builds and returns a struct containing read and write
+// NewRateLimiter builds and returns a struct containing read and write
```

Comment:
I needed this when creating the new OCI client for the e2e tests. It seemed wrong to cut and paste the code. This function seems to be a 'constructor', so I made it public.